src/results.ts

```diff
@@ -227,7 +227,7 @@ export class ParsingComponents implements ParsedComponents {
         } else {
             implySimilarTime(components, date);
             if (reference.timezoneOffset !== null) {
-                components.imply("timezoneOffset", -reference.instant.getTimezoneOffset());
+                components.imply("timezoneOffset", reference.timezoneOffset ?? -reference.instant.getTimezoneOffset());
```

Comment:
We don't need the fallback here. The `reference.timezoneOffset` should not be null, as it was checked above.